extensions/sql-database-projects/src/projectProvider/projectProvider.ts

```diff
@@ -168,4 +168,19 @@ export class SqlDatabaseProjectProvider implements dataworkspace.IProjectProvide
 		const projectUri = getDataWorkspaceExtensionApi().openSpecificProjectNewProjectDialog(projectType);
 		return projectUri;
 	}
+
+	/**
+ * Gets the list of .sql scripts contained in a project
+ * @param projectFilePath
```

Comment:
indent

Reply:
Huh, wonder why the linter didn't catch this